modular_skyrat/modules/extra_vv/code/extra_vv.dm

```diff
@@ -0,0 +1,43 @@
+/**
+ * EXTRA MOB VV
+ */
+/mob/vv_get_dropdown()
+	. = ..()
+	VV_DROPDOWN_OPTION(VV_HK_SEND_CRYO, "Send to Cryogenic Storage")
+	VV_DROPDOWN_OPTION(VV_HK_LOAD_PREFS, "Load Prefs Onto Mob")
+
+/mob/vv_do_topic(list/href_list)
+	. = ..()
+	if(href_list[VV_HK_SEND_CRYO])
+		if(!check_rights(R_SPAWN))
+			return
+		var/send_notice = tgui_alert(usr, "Add a paper notice about sending [name] into a cryopod?", "Leave a paper?", list("Yes", "No", "Cancel"))
+		if(send_notice != "Yes" && send_notice != "No")
+			return
+
+		//log/message
+		to_chat(usr, "Put [src] in cryopod.")
+		log_admin("[key_name(usr)] has put [key_name(src)] into a cryopod.")
+		var/msg = span_notice("[key_name_admin(usr)] has put [key_name(src)] into a cryopod from [ADMIN_VERBOSEJMP(src)].")
+		message_admins(msg)
+		admin_ticket_log(src, msg)
+
+		send_notice = send_notice == "Yes"
+		send_to_cryo(send_notice)
```

Comment:
I feel like, while you're here doing this, it could be a good idea to make this into a separate proc, like it's handled in the regular `/mob/vv_do_topic()`, should really be just making two more procs after this one, and it should be a lot more readable.

Reply:
I'm not sure if we should be continuing to add to proc overhead.

Reply:
This code isn't hot at all, proc overhead is irrelevant unless we're talking about genuinely hot code like atmos stuff.